.coveragerc

```diff
@@ -0,0 +1,5 @@
+[run]
+branch = True
+include = pyhap/*
+omit =
```

Comment:
Omit pyhap/accessories as well as it will be moved in a separate package, team or something else.

Reply:
See changes. 
49fd431